src/lang/lang_values.ml

```diff
@@ -212,6 +215,7 @@ and in_term =
   | Encoder of Encoder.format
   | List of term list
   | Tuple of term list
+  | Nothing
```

Comment:
Since `Nothing` is essentially a ground value, should we register it via `Lang.MkAbstract`? I can see arguments in both sides.

Reply:
I'll let you decide on that, I am not so familiar with abstract values. The only thing which makes this act weird is the subtyping `'a <: `'a?`, but I don't know if this is relevant here.

Reply:
I think it needs to be in the main language file since it is a global definition that should be clearly visible to all callers of this API.